holidays/countries/fiji.py

```diff
@@ -0,0 +1,182 @@
+         * [2024](https://web.archive.org/web/20250121185434/https://www.fiji.gov.fj/About-Fiji/Public-Holidays)
+         * [2025](https://web.archive.org/web/20250318092311/https://www.fiji.gov.fj/About-Fiji/Public-Holidays)
+
+    """
+
+    country = "FJ"
+    supported_categories = (PUBLIC, WORKDAY)
+    # %s (estimated).
+    estimated_label = "%s (estimated)"
+    # %s (observed).
+    observed_label = "%s (observed)"
+    # %s (observed, estimated).
+    observed_estimated_label = "%s (observed, estimated)"
+    # Act No. 13 of 2015
+    start_year = 2016
+
+    def __init__(self, islamic_show_estimated: bool = True, *args, **kwargs):
+        """
+        Args:
+            islamic_show_estimated:
+                Whether to add "estimated" label to Islamic holidays name
+                if holiday date is estimated.
+        """
+        ChristianHolidays.__init__(self)
+        HinduCalendarHolidays.__init__(self, cls=FijiHinduHolidays, show_estimated=True)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=FijiIslamicHolidays, show_estimated=islamic_show_estimated
+        )
+        kwargs.setdefault("observed_rule", SAT_SUN_TO_NEXT_MON)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_observed(self._add_new_years_day("New Year's Day"))
+
+        # Good Friday.
+        self._add_good_friday("Good Friday")
+
+        # Easter Saturday.
+        self._add_holy_saturday("Easter Saturday")
+
+        # Easter Monday.
+        self._add_easter_monday("Easter Monday")
+
+        # National Sports Day.
+        if self._year <= 2018:
+            self._add_holiday_last_fri_of_jun("National Sports Day")
+
+        if self._year <= 2022:
+            # Constitution Day.
+            self._add_observed(self._add_holiday_sep_7("Constitution Day"))
+
+        if self._year >= 2023:
+            # Girmit Day.
+            self._move_holiday(
+                self._add_holiday_may_14("Girmit Day"),
+                rule=ALL_TO_NEAREST_MON,
+                show_observed_label=False,
+            )
+
+        # Ratu Sir Lala Sukuna Day.
+        name = "Ratu Sir Lala Sukuna Day"
+        if self._year == 2023:
+            self._add_holiday_last_mon_of_may(name)
+        elif self._year >= 2024:
+            self._add_holiday_last_fri_of_may(name)
+
+        # Fiji Day.
+        self._add_holiday_oct_10("Fiji Day")
+
+        # Diwali.
+        self._add_observed(self._add_diwali_india("Diwali"))
+
+        # Christmas Day.
+        self._add_observed(self._add_christmas_day("Christmas Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)
+
+        # Boxing Day.
+        self._add_observed(self._add_christmas_day_two("Boxing Day"), rule=SAT_SUN_TO_NEXT_MON_TUE)
+
+        # Prophet Mohammed's Birthday.
+        self._populate_observed(self._add_mawlid_day("Prophet Mohammed's Birthday"))
```

Comment:
```suggestion
        for dt in self._add_mawlid_day("Prophet Mohammed's Birthday"):
            self._add_observed(dt)
```
@coderabbitai, thank you for this catch! :)